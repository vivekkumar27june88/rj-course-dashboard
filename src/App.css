:root {
    /* Colors */
    --rcd-bg-col: #dfe1eb;
    --rcd-sidenav-bg-col: #373b53;
    --rcd-card-bg-col: #ffffff;
    --rcd-text-secondary-col: #a6acbe;
    --rcd-text-primary-col: #000000;
    --rcd-icon-border-col: #ccc;

    /* Dimensions */
    --rcd-sidenav-width: 100px;
    --rcd-header-height: 94px; /* 50px + 27px + 27px */
}

.App {
    min-height: 100vh;
    width: 100%;
    background-color: var(--rcd-bg-col);
}
.main-container-1 {
    display: flex;
    flex-direction: row;
}

.main-container-2 {
    display: flex;
    flex-direction: column;
    width: calc(100vw - 100px);
    padding-left: var(--rcd-sidenav-width);
}
