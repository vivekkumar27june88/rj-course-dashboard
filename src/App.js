import React from 'react';
import './App.css';
import Header from './components/Header';
import Navbar from './components/Navbar';
import ContentView from './components/ContentView';

function App() {
    return (
        <main className="App">
            <div className="main-container-1">
                <Navbar />
                <div className="main-container-2">
                    <Header />
                    <ContentView />
                </div>
            </div>
        </main>
    );
}

export default App;
