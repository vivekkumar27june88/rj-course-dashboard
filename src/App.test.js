// import React from 'react';
// import { render } from '@testing-library/react';
// import App from './App';

// test('renders learn react link', () => {
//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
    const app = shallow(<App />);

    it('should render corretly', () => {
        expect(app).toMatchSnapshot();
        expect(app.find('.main-container-1').children().length).toBe(2);
    });
});
