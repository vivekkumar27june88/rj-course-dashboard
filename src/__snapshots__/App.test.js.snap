// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render corretly 1`] = `
<main
  className="App"
>
  <div
    className="main-container-1"
  >
    <Navbar />
    <div
      className="main-container-2"
    >
      <Header />
      <ContentView />
    </div>
  </div>
</main>
`;
